### PR TITLE
Added PayPal Acceptance Tests

### PR DESCRIPTION
**Pull request by clintonb**
OMFG WTFBBQ!!!!
@jimabramson @rlucioni

### CODE REVIEW

File: acceptance_tests/test_payment.py [hunk 1]
```diff
@@ -16,34 +16,135 @@ def setUp(self):
...
+        except (NoSuchElementException, TimeoutException):
+            # The PayPal form may be visible by default.
+            pass
+
+        # Wait for form to load
+        WebDriverWait(self.browser, 10).until(EC.presence_of_element_located((By.ID, 'login_email')))
+
+        # Log into PayPal
+        self.browser.find_element_by_css_selector('input#login_email').send_keys('clinton.blackburn@gmail.com')
+        self.browser.find_element_by_css_selector('input#login_password').send_keys('clintonblackburn')
```
Comment (rlucioni) [inline acceptance_tests/test_payment.py]: This should be configurable.

Comment (clintonb) [inline acceptance_tests/test_payment.py]: Updated.

**Comment by jimabramson** [comment]
:+1:

**Comment by rlucioni** [comment]
:tada: :tada: :tada:
:tada: :+1: :tada:
:tada: :tada: :tada:

**Pull request merged**